FizzBuzz/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*FizzBuzz
+ * A program that writes numbers from 1 to 100. Any number that is a multiple of three is replaced
+ * by the word 'Fizz', any number that is a multiple of five is replaced by the word 'Buzz', and any
+ * number that is a multiple of both three and five is replaced by the word 'FizzBuzz'.
+ */
+
+using System;
+
+namespace FizzBuzz
+{
+    class Program
+    {
+            static void Main(string[] args)
+            {
+                double ii;
```

Comment:
I assume you made this a double so that the division below works correctly - it should probably be an int though to more accurately represent its function, its an iterator from 1 - n. Cast it to double inline when you are performing calculations if needed.